source/manual/business-readiness-finder.html.md

```diff
@@ -0,0 +1,64 @@
+---
+title: EU exit business readiness finder
+section: Business readiness finder
+layout: manual_layout
+parent: "/manual.html"
+owner_slack: "#govuk-euexit-ready"
+last_reviewed_on: 2019-06-13
+review_in: 6 months
+---
+
+The [EU exit business readiness finder](https://www.gov.uk/find-eu-exit-guidance-business) is a special finder built to help businesses prepare for EU exit. A number of proprietary modifications have been made to the normal [finder-frontend](/apps/finder-frontend.html) model, which we've documented here.
+
+## Overview
+
+The full journey for the business readiness finder is made up of the following content items:
+
+1. The start page: [`/business-uk-leaving-eu`](https://www.gov.uk/business-uk-leaving-eu)
+2. The Q&A: [`/prepare-business-uk-leaving-eu`](https://www.gov.uk/prepare-business-uk-leaving-eu)
+3. The finder: [`/find-eu-exit-guidance-business`](https://www.gov.uk/find-eu-exit-guidance-business)
+4. The email signup page: [`/find-eu-exit-guidance-business/email-signup`](https://www.gov.uk/find-eu-exit-guidance-business/email-signup)
+
+### Start page
+
+A normal page which can be updated in Mainstream Publisher as you would any other start page. (No modifications).
+
+### Q&A
+
+The content item for the Q&A doesn't actually contain any details. The titles of the questions are defined in a YAML file in [finder frontend](https://github.com/alphagov/finder-frontend/blob/3d7f25ddca4bedd9d9fb750fb1d651964cf2a34b/lib/prepare_business_uk_leaving_eu.yaml). However the body of the question, for example the options if the question is a checkbox, are read from the content item of the finder itself (https://www.gov.uk/api/content/find-eu-exit-guidance-business). Finder frontend searches the content item for a facet "key" that matches the question, e.g. `sector_business_area` to find the question content.
```

Comment:
Could the https://www.gov.uk/api/content/find-eu-exit-guidance-business be a link?
eg. `[content item of the finder](https://www.gov.uk/api/content/find-eu-exit-guidance-business) itself.`

Reply:
Will fix this 👍 thanks. Didn't know we had a Support app! This is definitely worth documenting:

> It currently gives an option to request that the content is pinned, so this part should be removed when the pinning behaviour is removed.

Are there any other docs I can read to try to understand the Support app?

NB, I see this when I visit https://support.integration.publishing.service.gov.uk/eu_exit_business_readiness_request/new:
![image](https://user-images.githubusercontent.com/5111927/59615796-112fca80-911b-11e9-88d9-efa538e07c41.png)

(what permissions do I need to grant myself?)

Reply:
I think [the README](https://github.com/alphagov/support) is the most useful.
You need to give yourself the `content_requester` permission in Signon.

Regarding the Business finder form, it's also worth noting that:
- facet values come from Publishing API
- the questions relating to facets are hardcoded and there is one question gives yes/no option instead of the facet values. This was reviewed by a content designer who advised to keep it this way.

Reply:
> the questions relating to facets are hardcoded and there is one question that gives yes/no option instead of the facet values

Can I just check you mean [the second page of the Q&A](https://www.gov.uk/prepare-business-uk-leaving-eu?page=2) where we reduce it to a Yes/No question instead of listing all the possible business activities?

Reply:
I meant this question of the form:
<img width="575" alt="Screen Shot 2019-06-18 at 13 17 34" src="https://user-images.githubusercontent.com/38078064/59681298-80fd8e00-91cb-11e9-9c3a-d063bb48edb6.png">
where corresponding facet values are _EU citizens, No EU citizens_.